templates/configmap.yaml

```diff
@@ -94,7 +95,7 @@ data:
     [astronomer]
 {{- if .Values.webserver.jwtSigningCertificateSecretName }}
     jwt_signing_cert = {{ template "airflow_webserver_jwt_cert_path" . }}
-    jwt_audience = {{ include "airflow_url" . }}
+    jwt_audience = {{ include "deployments_host" . }}
```

Comment:
This needs to change - see https://github.com/astronomer/houston-api/pull/211#pullrequestreview-352285616